schema-engine/connectors/sql-schema-connector/src/datamodel_calculator/context/flavour/postgresql.rs

```diff
@@ -24,5 +24,21 @@ impl super::IntrospectionFlavour for PostgresIntrospectionFlavour {
                 });
             }
         }
+
+        for check_constraint in ctx.sql_schema.check_constraints() {
+            let check_constraint = CheckConstraint {
```

Comment:
I'd use something else here, e.g. the `ModelAndConstraint` which we already have in place.